<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FurryViewer2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
border-image: url(:/back.png) 0 0 0 0 stretch stretch;
}
</string>
   </property>
   <widget class="GLWidget" name="GLwidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>451</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonpath">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLabel {
    border: 0.5px solid green;
    border-radius: 4px;
    padding: 2px;
   }</string>
    </property>
    <property name="text">
     <string>TRANSLATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLabel {
    border: 0.5px solid green;
    border-radius: 4px;
    padding: 2px;
   }</string>
    </property>
    <property name="text">
     <string>ROTATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>395</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLabel {
    border: 0.5px solid green;
    border-radius: 4px;
    padding: 2px;
   }</string>
    </property>
    <property name="text">
     <string>SCALE</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>480</y>
      <width>142</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>FILE NAME</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>NUMBER OF VERTICES</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>NUMBER OF RIBS</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>480</y>
      <width>241</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="path_line">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 221, 218);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="number_vert">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 221, 218);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="number_ribs">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 221, 218);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>440</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLabel {
    border: 0.5px solid green;
    border-radius: 4px;
    padding: 2px;
   }</string>
    </property>
    <property name="text">
     <string>PROJECTION</string>
    </property>
   </widget>
   <widget class="QComboBox" name="type_of_projection">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>central</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>parallel</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="background_color_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>650</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>BACK COLOR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>565</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLabel {
    border: 0.5px solid green;
    border-radius: 4px;
    padding: 2px;
   }</string>
    </property>
    <property name="text">
     <string>RIBS TYPE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ribstype">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>560</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>solid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dotted</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="color_of_rebra_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>RIBS COLOR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>605</y>
      <width>113</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLabel {
    border: 0.5px solid green;
    border-radius: 4px;
    padding: 2px;
   }</string>
    </property>
    <property name="text">
     <string>RIBS THICKNESS</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ribthick">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>600</y>
      <width>65</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="verttype">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>none</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>circle</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="color_of_vert_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>650</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>VERTICES COLOR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>565</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLabel {
    border: 0.5px solid green;
    border-radius: 4px;
    padding: 2px;
   }</string>
    </property>
    <property name="text">
     <string>VERTICES TYPE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>605</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLabel {
    border: 0.5px solid green;
    border-radius: 4px;
    padding: 2px;
   }</string>
    </property>
    <property name="text">
     <string>VERTICES SIZE</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="vertsize">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>600</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_screen_bmp">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>510</y>
      <width>122</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>BMP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_screen_gif">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>610</y>
      <width>122</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>GIF</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x_trans">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.020000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y_trans">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>169</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.020000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="z_trans">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>199</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.020000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x_rot">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>279</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y_rot">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>309</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="z_rot">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scale_value">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>504</x>
      <y>143</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>170</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>200</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>280</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>310</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>341</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>30</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="label_x">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_z">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_y">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_x_r">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_z_r">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_y_r">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="scale_label">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_screen_jpg">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>560</y>
      <width>122</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>JPEG</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
